src/ngraph/descriptor/tensor.hpp

```diff
@@ -16,55 +16,69 @@
 
 #pragma once
 
-#include <iostream>
 #include <memory>
-#include <vector>
+#include <string>
 
+#include "ngraph/descriptor/tensor.hpp"
 #include "ngraph/shape.hpp"
 #include "ngraph/type/element_type.hpp"
 
 namespace ngraph
 {
     class Node;
 
-    namespace element
-    {
-        class Type;
-    }
-
     namespace descriptor
     {
-        class TensorView;
-        class Tensor;
-    }
-}
+        namespace layout
+        {
+            class TensorViewLayout;
```

Comment:
Is TensorViewLayout going to be renamed TensorLayout in a separate PR?

Reply:
It might merge in to Tensor as well since there's only one per tensor.